.travis.yml

```
@@ -1,39 +1,43 @@
-language: julia
+# For learning how to customize the CI check out
+# Continous Integration:
+# https://docs.travis-ci.com/user/languages/julia
+# https://github.com/marketplace/travis-ci
+# Code Coverage:
+# https://github.com/JuliaCI/Coverage.jl
+# Documentation:
+# https://github.com/JuliaDocs/Documenter.jl
 
+language: julia
 os:
-  - osx
   - linux
-
+  - osx
+  - windows
 julia:
   - 1.0
-  - 1.1
   - 1.2
   - nightly
-
-# # Uncomment the following lines to allow failures on nightly julia
-# # (tests will run but not make your overall status red)
-# matrix:
-#   allow_failures:
-#   - julia: nightly
-
+codecov: true
 notifications:
   email: false
-
-#script: # the default script is equivalent to the following
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --project -e 'using Pkg; Pkg.build(); Pkg.test(; coverage=true)';
-
-after_success:
-  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
-#  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())';
-
 jobs:
+  allow_failures:
+    - julia: nightly
+  fast_finish: true
   include:
-    - stage: "Documentation"
+    - os: windows
+      arch: x86
       julia: 1.0
+    - os: windows
+      arch: x86
+      julia: 1.2
+    - os: windows
+      arch: x86
+      julia: nightly
+    - stage: "Documentation"
       os: linux
+      julia: 1.2
```

Test documentation on the latest Julia release
```suggestion
      julia: 1
```